inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaPropertyElement.java

```diff
@@ -71,30 +69,31 @@ class JavaPropertyElement extends AbstractJavaElement implements PropertyElement
         this.visitorContext = visitorContext;
     }
 
-    /**
-     * Default constructor.
-     *
-     * @param declaringElement   The declaring element
-     * @param rootElement        The element
-     * @param annotationMetadata The annotation metadata
-     * @param name               The name
-     * @param type               The type
-     * @param readOnly           Whether it is read only
-     * @param visitorContext     The java visitor context
-     * @deprecated Use {@link #JavaPropertyElement(ClassElement, Element, AnnotationMetadata, String, ClassElement, boolean, JavaVisitorContext)} instead
-     */
-    @SuppressWarnings("DeprecatedIsStillUsed") // used by openapi processor
-    @Deprecated
-    JavaPropertyElement(
-            ClassElement declaringElement,
-            ExecutableElement rootElement,
-            AnnotationMetadata annotationMetadata,
-            String name,
-            ClassElement type,
-            boolean readOnly,
-            JavaVisitorContext visitorContext) {
-        this(declaringElement, (Element) rootElement, annotationMetadata, name, type, readOnly, visitorContext);
-    }
+//    /**
+//     * Default constructor.
+//     *
+//     * @param declaringElement   The declaring element
+//     * @param rootElement        The element
+//     * @param annotationMetadata The annotation metadata
+//     * @param name               The name
+//     * @param type               The type
+//     * @param readOnly           Whether it is read only
+//     * @param visitorContext     The java visitor context
+//     * @deprecated Use {@link #JavaPropertyElement(ClassElement, Element, AnnotationMetadata, String, ClassElement, boolean, JavaVisitorContext)} instead
+//     */
+//    @SuppressWarnings("DeprecatedIsStillUsed") // used by openapi processor
```

Comment:
We should not leave it commented it out. Does the OpenAPI processor use it? If the OpenAPI processor uses it, revert this change so we can merge this Pr.

Reply:
That was unintentional. I meant to delete it.